### PR TITLE
feat(admin-api+admin-ui): option to uninstall applications from node

### PR DESCRIPTION
**Pull request by frdomovic**
# feat(admin-api+admin-ui): option to uninstall applications from node

## Summary
Added new api endpoint for app uninstall + code for ctx manager to delete it from node.
Updated applications page to display proper values (previously missing metadata)
Option to uninstall application.

Application can only be uninstalled if no context is using it.

Fixes # Issue #524 
## Test plan

https://github.com/user-attachments/assets/7d92c25a-341e-4e76-bcf4-d571f7c4a876

### CODE REVIEW

File: node-ui/build/assets/main-DzpClS_l.js
Comment (miraclx) [inline node-ui/build/assets/main-DzpClS_l.js:1]: should we commit this?

**Review by miraclx — APPROVED** [review]

File: node-ui/src/pages/Applications.tsx [hunk 1]
```diff
@@ -114,74 +123,127 @@ export default function ApplicationsPage() {
...
+  useEffect(() => {
     setApps();
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const uninstallApplication = async () => {
+    const contextList = (
+      await apiClient(showServerDownPopup).node().getContexts()
+    ).data?.contexts;
+    if (contextList?.length !== 0) {
```
Comment (alenmestrov) [inline node-ui/src/pages/Applications.tsx]: is it possible that this request returns error?

Comment (frdomovic) [reply node-ui/src/pages/Applications.tsx]: Updated to handle error also

File: node-ui/build/assets/main-DzpClS_l.js
Comment (frdomovic) [reply node-ui/build/assets/main-DzpClS_l.js:1]: Updated, not included

**Pull request merged**